tests/test_views_hello.py

```diff
@@ -23,6 +24,17 @@ def test_returns_user_id_if_authenticated(self):
         response = self.app.get('/', headers=self.headers)
         self.assertEqual(response.json['user']['id'], self.principal)
 
+    def test_returns_user_principals_if_authenticated(self):
+        group_url = '/buckets/beers/groups/users'
+        group = MINIMALIST_GROUP.copy()
+        group['data']['members'].append(self.principal)
+        self.app.put_json('/buckets/beers', MINIMALIST_BUCKET, headers=self.headers)
+        self.app.put_json(group_url, group, headers=self.headers)
+        response = self.app.get('/', headers=self.headers).json['user']['principals']
+        principals = ('system.Everyone', 'system.Authenticated',
+                      group_url, self.principal.split(':')[1])
```

Comment:
Note that we also remove the other one: https://github.com/Kinto/kinto/blob/master/kinto/core/authorization.py#L64